clang/test/CodeGenCXX/debug-info-alias.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang -g -std=c++11 -S -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang -ggdb -std=c++11 -S -emit-llvm %s -o - | FileCheck %s
```

Comment:
Do we now lose a tiny bit of coverage for `-glldb`? I assume this change is to make sure the test passes on platforms that are tuned for `sce`? Maybe we should just add `-gno-template-alias` instead?